tests/test_tui_api.py

```diff
@@ -5,12 +5,9 @@
 from ansys.fluent.core.services.datamodel_tui import TUIMenu
 
 
-@pytest.mark.skip("randomly failing due to missing transcript capture")
+@pytest.mark.skip("Failing in github")
 def test_report_system_proc_stats_tui(new_solver_session, capsys) -> None:
-    new_solver_session.transcript.start()
-    # Issue: Transcript missing for the first TUI command
-    new_solver_session.solver.tui.report.system.sys_stats()
-    new_solver_session.solver.tui.report.system.sys_stats()
+    new_solver_session.tui.report.system.sys_stats()
     captured = capsys.readouterr()
     assert "CPU" in captured.out
```

Comment:
Is it possible the output ended up in stderr rather than stdout for some reason? This will check both of them:
```suggestion
    assert "CPU" in str(captured)
```